python/UniverseWrapper.cpp

```diff
@@ -72,20 +72,14 @@ namespace {
     const Building*         GetBuildingP(const Universe& universe, int id)
     { return ::GetBuilding(id).operator->(); }
 
-    std::vector<int>        ObjectIDs(const Universe& universe)
-    { return Objects().FindObjectIDs(); }
-    std::vector<int>        FleetIDs(const Universe& universe)
-    { return Objects().FindObjectIDs<Fleet>(); }
-    std::vector<int>        SystemIDs(const Universe& universe)
-    { return Objects().FindObjectIDs<System>(); }
-    std::vector<int>        FieldIDs(const Universe& universe)
-    { return Objects().FindObjectIDs<Field>(); }
-    std::vector<int>        PlanetIDs(const Universe& universe)
-    { return Objects().FindObjectIDs<Planet>(); }
-    std::vector<int>        ShipIDs(const Universe& universe)
-    { return Objects().FindObjectIDs<Ship>(); }
-    std::vector<int>        BuildingIDs(const Universe& universe)
-    { return Objects().FindObjectIDs<Building>(); }
+    template<typename T>
+    std::vector<int> ObjectIDs(const Universe& universe)
+    {
+        std::vector<int> result;
```

Comment:
probably worth reserving space before repeatedly pushing a known number of items into container

Reply:
Done